qiskit_dynamics/solvers/solver_functions.py

```diff
@@ -84,8 +95,15 @@ def solve_ode(
     - ``scipy.integrate.solve_ivp`` - supports methods
       ``['RK45', 'RK23', 'BDF', 'DOP853', 'Radau', 'LSODA']`` or by passing a valid
       ``scipy`` :class:`OdeSolver` instance.
-    - ``jax.experimental.ode.odeint`` - accessed via passing
-      ``method='jax_odeint'``.
+    - ``'RK4'``: A fixed-step 4th order Runge-Kutta solver.
+      Requires additional kwarg ``max_dt``, indicating the maximum step
+      size to take. This solver will break integration periods into even
+      sub-intervals no larger than ``max_dt``, and step over each sub-interval
+      using the standard 4th order Runge-Kutta integration rule.
+    - ``'jax_RK4'``: JAX-version of ``'RK4'``, implemented using JAX
```

Comment:
"JAX looping logic" is pretty vague for a method description. Probably either just something like "JAX backend implementation of RK4 method." or could specifically say if it automatically uses jit/autograd in the method or not.

Reply:
Changed it to your suggestion. We don't jit anything automatically in dynamics so I won't say anything about this.